### DISCUSSION TITLE
amdgpu fan speed is always 0

### DISCUSSION THREAD

**Discussion by Rom888** [AdityaGarg8/mbp-16.x-ubuntu-kernel#89] [Q&A] [answered]
Hi!
I see that amdgpu fan speed is always 0 even if amdgpu is really loaded and cpu funs work hard.
Is it possible to activate it?

```
macbook pro, 16 inch, 2019
Intel Core i9 2.3g

Intel UHD graphics 630
AMD radeon pro 5500M
```

**Comment by Redecorating** [comment, marked as answer]
Hi, your model has two fans, both are controlled by applesmc. I think the amdgpu fan doesn't actually exist, it's just there because other computers have fans attached to their gpus specifically

**Reply by Redecorating** [reply]
because i remember there were pictures and i think it's interesting: you can actually see what it looks like inside here https://www.ifixit.com/Teardown/MacBook+Pro+16-Inch+2019+Teardown/128106#s249744

![image](https://github.com/t2linux/T2-Debian-and-Ubuntu-Kernel/assets/69827514/26ce7139-8331-4a12-a631-90e548a8b273)
![image](https://github.com/t2linux/T2-Debian-and-Ubuntu-Kernel/assets/69827514/c287dbfb-6320-42b7-a4b2-103de098f679)

under the big copper heat pipe there's both the gpu and cpu, and both fans try to cool the heat pipe

**Reply by Rom888** [reply]
Hi, thanks for the clarification!
I thought it exists, because 'sensor' shows it:

$ sensor
```
amdgpu-pci-0300
Adapter: PCI adapter
vddgfx:        6.00 mV 
fan1:           0 RPM  (min =    0 RPM, max = 4950 RPM)
edge:         +57.0°C  (crit = +110.0°C, hyst = -273.1°C)
                       (emerg = +115.0°C)
junction:     +57.0°C  (crit = +105.0°C, hyst = -273.1°C)
                       (emerg = +110.0°C)
mem:           +0.0°C  (crit = +150.0°C, hyst = -273.1°C)
                       (emerg = +155.0°C)
PPT:           4.00 W  (cap =  50.00 W)
...
```
